packages/CodingStandard/tests/Fixer/LineLength/BreakArrayListFixer/wrong/wrong2.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+class SomeClass
+{
+    public function someFunction()
+    {
+        $result = ['loooooooooooooooooooooooooooooooongArraaaaaaaaaaay', 'loooooooooooooooooooooongArraaaaaaaaaaaaaaay'];
```

Comment:
Maybe we can use some Lorem Ipsum text to avoid this?

Reply:
We could use something else, but "long array" message should remain. So it's clear what line is about to be fixed here.